tests/WE2E/test_configs/wflow_features/config.get_from_NOMADS_ics_FV3GFS_lbcs_FV3GFS.yaml

```diff
@@ -5,20 +5,19 @@ metadata:
     (from which ICs and LBCs will be derived).
 user:
   RUN_ENVIR: community
+platform:
+  EXTRN_MDL_DATA_STORES: nomads hpss aws
 workflow:
   CCPP_PHYS_SUITE: FV3_GFS_2017_gfdlmp
-  DATE_FIRST_CYCL: '2020082600'
-  DATE_LAST_CYCL: '2020082600'
+  PREDEF_GRID_NAME: RRFS_CONUS_25km
+  DATE_FIRST_CYCL: date --utc --date="2 days ago" +%Y%m%d00
+  DATE_LAST_CYCL: date --utc --date="2 days ago" +%Y%m%d00
```

Comment:
NICE!

Reply:
I originally wanted to handle this in the python-workflow but it ended up to be unnecessarily complicated so left it to WE2E scripts as a temporary solution. I think once the `run_WE2E` scripts are converted to python, templating the WE2E config files themselves with jinja2 may have some benefits.

Reply:
I am working on that solution now!! (Templating with jinja2 in the config files)